lib/friendly-errors/formatters/missing-css-file.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * This file is part of the Symfony Webpack Encore package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+'use strict';
+
+const chalk = require('chalk');
+
+function formatErrors(errors) {
+    if (errors.length === 0) {
+        return [];
+    }
+
+    let messages = [];
+
+    messages.push(
+        chalk.red('Module build failed: Module not found:')
+    );
+    for (let error of errors) {
+        messages.push(error.file + ' contains a reference to the file ' + error.ref + '.');
```

Comment:
A bit of nitpicking but you can use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) for that kind of thing, for instance:

```js
message.push(`${error.file} contains a reference to the file ${error.ref}.`);
```